### ISSUE TITLE
Modals & Expose should have consistent close buttons for a11y reasons

### ISSUE THREAD

**Issue by polyester** [plone/mockup#8]
Some modals and popups have a "close" button, others don't. Even on those that have it, the button is on the top left (for sighted users), before the first form element (for screenreader users), meaning they would have to go 'back', but there is no indication this is what they should do..

Clues like visual darkening (Expose) don't work for screenreader users, so they also don't see the notion they should 'click somewhere on the dark area' to get out of the modal.

Ideally, a close button should be placed at the end of the DOM for the modal. If there is alreay one on the topleft, this can probably be classed 'hiddenStructure' so no duplicate buttons appear for sighted users.
